draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -717,37 +717,37 @@ A RADIUS/DTLS server using the 5-tuple method MUST track ongoing DTLS sessions f
 
 Note that this 5-tuple is independent of IP address version (IPv4 or IPv6).
 
-Each 5-tuple points to a unique session entry, which usually contains the following information:
+Each 5-tuple points to a unique connection entry, which usually contains the following information:
 
-DTLS Session:
-: Any information required to maintain and manage the DTLS session.
+DTLS Connection:
+: Any information required to maintain and manage the DTLS connection.
 
 DTLS Data:
-: An implementation-specific variable that may contain information about the active DTLS session.
+: An implementation-specific variable that may contain information about the active DTLS connection.
 This variable may be empty or nonexistent.
 
-: This data will typically contain information such as idle timeouts, session lifetimes, and other implementation-specific data.
+: This data will typically contain information such as idle timeouts, connection lifetimes, and other implementation-specific data.
 
-#### Session Opening and Closing
+#### Connection Opening and Closing
 
-Session tracking is subject to Denial-of-Service (DoS) attacks due to the ability of an attacker to forge UDP traffic.
+Connection tracking is subject to Denial-of-Service (DoS) attacks due to the ability of an attacker to forge UDP traffic.
 RADIUS/DTLS servers SHOULD use the stateless cookie tracking technique described in {{!RFC6347, Section 4.2.1}}.
-DTLS sessions SHOULD NOT be tracked until a ClientHello packet has been received with an appropriate Cookie value.
-Server implementation SHOULD have a way of tracking DTLS sessions that are partially set up.
-Servers MUST limit both the number and impact on resources of partial sessions.
+DTLS connections SHOULD NOT be tracked until a ClientHello packet has been received with an appropriate Cookie value.
+Server implementation SHOULD have a way of tracking DTLS connections that are partially set up.
+Servers MUST limit both the number and impact on resources of partial connections.
 
-Sessions (both 5-tuple and entry) MUST be deleted when the DTLS session is closed for any reason.
-When a session is deleted due to it failing security requirements, the DTLS session MUST be closed, any TLS session resumption parameters for that session MUST be discarded, and all tracking information MUST be deleted.
+Connections (both 5-tuple and entry) MUST be deleted when the DTLS connection is closed for any reason.
```

Comment:
Deleting a conection doesn't really make sense. Either it must delete data related to the connection or the session data (so session resumption is no longer possible)